### ISSUE TITLE
try deleting subfolders of `src` folder

### ISSUE THREAD

**Issue by dlaehnemann** [hzi-bifo/mdasim#8]
The subfolders `src/lib` and `src/mdasim` seem to just contain old versions of the main code files in the main `src` folder. So try deleting them and double-check if the tool still compiles and works. This cleanup will avoid future confusion.

**Comment by VSack** [comment]
Trial and error: removing the subfolders does not work just like that.

If we take a look at the Makefile, we will find the lines
```
LIBSRCS = $(sort $(wildcard $(SRCDIR)/lib/*.C))
```
and 
```
HYDASRCS = $(sort $(wildcard $(SRCDIR)/$(PACKAGENAME)/*.C))
```
I don't know yet what these files are doing since they look awfully similar to the ones in `src` but I cannot just delete them without destroying the build process.

**Comment by VSack** [comment]
Closed by 08eb6314a8c61ef42a471c1030bc49d132dd0783
